README.md

```diff
@@ -28,6 +30,7 @@ Below you'll find the different tracks, but we recommend that all participants f
 - [Frontend Challenges](/frontend/README.md)
 - [Backend Challenges](/backend/README.md)
 - [Stretch Challenges](/STRETCH.md)
+- [Glossary](./GLOSSARY.md)
```

Comment:
It's better to use absolute URL because those will keep working even if you move files around :). There are then also not dependent on from "where" they look at the file.